### PR TITLE
figma,svelte,tailwind,next, added

### PR DESCRIPTION
**Pull request by RockingSNP**
**### _Sveltekit not found: rest is sorted.._**

### CODE REVIEW

File: data/project.data.ts [hunk 1]
```diff
@@ -23,14 +23,22 @@ interface ProjectData {
...
+   firebase: {
     title: 'Firebase',
     icon: 'firebase-plain'
   },
   javascript: {
     title: 'JavaScript',
     icon: 'javascript-plain'
   },
+  next: {
+    title: 'next',
```
Comment (sammaji) [inline data/project.data.ts]:
```suggestion
    title: 'NextJS',
```

File: data/project.data.ts [hunk 2]
```diff
@@ -39,18 +47,29 @@ const TECH_STACK = {
     title: 'React',
     icon: 'react-original'
   },
+  svelte: {
+    title: 'svelete',
```
Comment (sammaji) [inline data/project.data.ts]:
```suggestion
    title: 'Svelete',
```

**Review by sammaji — CHANGES_REQUESTED** [review]
Make sure `title` values starts with uppercase. Also change js to JS
eg, threeJs --> ThreeJS

**Comment by RockingSNP** [comment]
Ok it's revised

File: data/project.data.ts [hunk 3]
```diff
@@ -23,14 +23,22 @@ interface ProjectData {
 
 /* A full list of technologies I have worked with, in alphabetical order */
 const TECH_STACK = {
-  firebase: {
+  figma: {
+    title: 'Figma',
+    icon: 'figma-plain-colored'
```
Comment (sammaji) [inline data/project.data.ts]:
```suggestion
    icon: 'figma-plain'
```

File: data/project.data.ts [hunk 4]
```diff
@@ -39,18 +47,29 @@ const TECH_STACK = {
     title: 'React',
     icon: 'react-original'
   },
+  svelte: {
+    title: 'Svelete',
+    icon: 'svelte-plain'
+  },
+  tailwind: {
+    title: 'Tailwind',
+    icon: 'tailwindcss-original-wordmark colored'
```
Comment (sammaji) [inline data/project.data.ts]:
```suggestion
    icon: 'tailwindcss-original'
```

**Review by sammaji — CHANGES_REQUESTED** [review]
for `icon` values remove "-color" or " color" or "-wordmark" as I don't want colored or word-marked versions.
eg, figma-plain-wordmark color --> figma-plain

File: data/project.data.ts (hunk 3 above)
Comment (RockingSNP) [reply data/project.data.ts]: on it

**Review by RockingSNP — COMMENTED** [review]
done

**Review by sammaji — APPROVED** [review]
Looks good

**Pull request merged**